### DISCUSSION TITLE
Is the steam deck docking station required to install Jovian-NixOS?

### DISCUSSION THREAD

**Discussion by dnkmmr69420** [Jovian-Experiments/Jovian-NixOS#180] [Q&A] [answered]

**Comment by samueldr** [comment, marked as answer]
No.

Though you will need some way to use a physical keyboard for the initial NixOS install, possibly... Though maybe the graphical NixOS installer can help there?

**Comment by dnkmmr69420** [comment]
Yeah I will use the graphical nixos installer just like my other devices. I wasn't sure if drivers are required to use any button, joystick, or whatever else there is.

**Comment by samueldr** [comment]
None, without being triggered via software, it acts like a keyboard/mouse combo with very limited keyboard input.

**Comment by dnkmmr69420** [comment]
I think we should have this answered in the FAQ




Is the docking station required to install jovian-NixOS?
